test/common/formatter/substitution_formatter_test.cc

```diff
@@ -3327,17 +3329,17 @@ TEST(SubstitutionFormatterTest, StructFormatterOmitEmptyTest) {
 
   ProtobufWkt::Struct key_mapping;
   TestUtility::loadFromYaml(R"EOF(
-      test_key_filter_state: '%FILTER_STATE(nonexistent_key)%'
-      test_key_req: '%REQ(nonexistent_key)%'
-      test_key_res: '%RESP(nonexistent_key)%'
-      test_key_dynamic_metadata: '%DYNAMIC_METADATA(nonexistent_key)%'
+      test_key_root:
```

Comment:
I assume that `test_key_root` was added to create a nested field. But it also removed a test case for empty non-nested field. Is this correct?

Reply:
Correct. I have now created a new test, so the existing test works as before.